motioneye/uploadservices.py

```diff
@@ -1290,9 +1290,9 @@ def test_access(camera_id, service_name, data):
     return service.test_access()
 
 
-def update(camera_id, service_name, settings):
+def update(camera_id, service_name, new_settings):
     service = get(camera_id, service_name)
-    service.load(settings)
+    service.load(new_settings)
```

Comment:
This would break the (single) usage of the function, since arguments are passed as `**kwargs`. So the variable name would need to be changed in the function call as well: https://github.com/motioneye-project/motioneye/blob/0b7a6b8/motioneye/config.py#L1040-L1052
We could use `upload_settings` to match the original name of the passed variable, or switch to `options`. This is to not reuse `from motioneye import settings`.